package.json

```diff
@@ -87,26 +93,33 @@
     "@typescript-eslint/parser": "^4.28.3",
     "@webcomponents/custom-elements": "^1.4.3",
     "@webcomponents/webcomponents-platform": "^1.0.1",
+    "autoprefixer": "^10.3.1",
     "browser-sync": "^2.27.4",
     "chokidar-cli": "^2.1.0",
+    "clean-css-cli": "^5.3.0",
     "concurrently": "^6.2.0",
     "copyfiles": "^2.4.1",
+    "cssnano": "^5.0.7",
     "eslint": "^7.30.0",
     "eslint-plugin-editorconfig": "^3.0.2",
     "eslint-plugin-import": "^2.23.4",
     "eslint-plugin-sonarjs": "^0.9.1",
     "foreach-cli": "^1.8.1",
+    "html-minifier": "^4.0.0",
     "husky": "^7.0.1",
     "intersection-observer": "^0.12.0",
     "jest": "^27.0.6",
     "less": "^4.1.1",
     "lintspaces-cli": "^0.7.1",
-    "postcss": "^8.3.5",
+    "postcss": "^8.3.6",
+    "postcss-cli": "^8.3.1",
     "promise-polyfill": "^8.2.0",
     "rimraf": "^3.0.2",
     "semantic-release": "^17.4.4",
     "smoothscroll-polyfill": "^0.4.4",
+    "style-loader": "^3.2.1",
```

Comment:
Do we need?

Reply:
Sorry, I didn't remove unused dependencies from package.json.